include/parser_interface.hpp

```diff
@@ -39,6 +39,27 @@ class ParserInterface
         return types::DbusVariantType();
     }
 
+    /**
+     * @brief API to write keyword's value on hardware.
+     *
+     * This virtual method is created to achieve runtime polymorphism for
+     * hardware writes on VPD of different types. This virtual method can be
+     * redefined at derived classess to implement write according to the type of
+     * VPD.
+     *
+     * @param[in] i_paramsToWriteData - Data required to perform write.
+     *
+     * @throw Throws exception.
```

Comment:
you could add a NOTE that says  "May throw exception depending on the specific implementation of the derived methods, so caller needs to handle exception for this.

Reply:
done